packages/merkle-distributor/src/CloudflareKVHelper.ts

```diff
@@ -0,0 +1,82 @@
+// This file contains some helper functions to wrap cloudflare Key Value store.
+
+const nodeFetch = require("node-fetch");
+const assert = require("assert");
```

Comment:
nit: import?

Reply:
done! I had to install `@types/node-fetch` but it worked.